### ISSUE TITLE
Automatically manipulate CSP to allow HUD to work

### ISSUE THREAD

**Issue by psiinon** [zaproxy/zap-hud#42]
The HUD extension should detect CSP and either diable it, or ideally just manipulate it so that the HUD will be always work.
This should be done automatically when the HUD is enabled.
Even if we can manipulate the CSP maybe we should have an option to just strip it out in case the manipulation doesnt work or breaks other things?

**Comment by JavanXD** [comment]
Yes you could add `Content-Security-Policy: script-src 'nonce-EDNnf03nceIOfn39fn3e9h3sdfa'`to the CSP.
And include it like:
```html
<script nonce="EDNnf03nceIOfn39fn3e9h3sdfa" src="//zap/hud.js" defer></script>
```
